pkgs/applications/networking/instant-messengers/bitlbee-facebook/default.nix

```diff
@@ -0,0 +1,27 @@
+{ fetchurl, fetchFromGitHub, stdenv, bitlbee, autoconf, automake, libtool, pkgconfig, glib, json_glib }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+  name = "bitlbee-facebook";
```

Comment:
Please add version. Where version is the date of the commit you are fetching from source. In the format `YYYY-MM-DD`.